tests/ApiManifest.Tests/OpenAIPluginManifestTests.cs

```diff
@@ -0,0 +1,395 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using Microsoft.OpenApi.ApiManifest.OpenAI;
+using System.Text.Json;
+
+namespace Microsoft.OpenApi.ApiManifest.Tests;
+
+public class OpenAIPluginManifestTests
+{
+    [Fact]
+    public void LoadOpenAIPluginManifest()
+    {
+        var json = @"{
```

Comment:
we should consider using string literals instead `"""` so we get an easier time reading things